wiring/inc/spark_wiring_thread.h

```diff
@@ -67,6 +67,7 @@ class Thread
     mutable wiring_thread_fn_t *wrapper = NULL;
     os_thread_fn_t func_ = NULL;
     void* func_param_ = NULL;
+    bool started_ = false;
```

Comment:
doesn't this need to be volatile since it's accessed by multiple threads without any memory guard?